library/fsharp

```diff
@@ -1,14 +1,14 @@
 Maintainers: Dave Curylo <dave@curylo.org> (@ninjarobot),
              Steve Desmond <steve@stevedesmond.ca> (@stevedesmond-ca)
 GitRepo: https://github.com/fsprojects/docker-fsharp.git
-GitCommit: 2ffa0ffb79c1607617c28130f981e83f55e6cbc5
+GitCommit: 6ced4c004bac9d5a024eefbc1d90219c78ce784b
 
-Tags: latest, 10, 10.6, 10.6.0
+Tags: latest, 10, 10.7, 10.7.0
```

Comment:
I'm confused by the version numbers. The image uses `FSHARP_VERSION=10.2.3`; how does that correspond to `10.6` and `10.7`?

Reply:
Interesting, thanks for the added detail! :+1: :heart: